version/version.go

```diff
@@ -1,51 +0,0 @@
-// Copyright 2022 Evmos Foundation
```

Comment:
Why is this deleted?

Reply:
@GAtom22 will be added in another PR.

Reply:
But why is it removed? Was it breaking something?